lib/land/tracker.rb

```diff
@@ -260,7 +260,7 @@ def request_domain
     end
 
     def new_visit?
-      @visit_id.nil? || Land.config.new_visit_reasons.map{ |reason| send(reason.to_sym) }.any?
+      Land::Visit.find_by(@visit_id).nil? || @visit_id.nil? || Land.config.new_visit_reasons.map{ |reason| send(reason.to_sym) }.any?
```

Comment:
```suggestion
      Land::Visit.find_by(@visit_id).nil? 
```

Here we actually only want to set a new visit if it doesn't exist, the front end cookies are responsible for expiring the visit (cookie lifetime or new query params)